source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+Another way of looking at this is to consider the goals for your mobile deployment.
+
+For example, do you want the simplest, fastest deployment possible?
+
+We have an option for that! Using the publicly available Mattermost mobile apps on the Google and Apple app stores can reduce a deployment to minutes.
+
+Better yet, this model can be rolled out to thousands of users in many different countries and languages?
+
+Of course, large enterprise deployments will take more time, and the overall goals for your deployment will depend on many factors.
+
+We recommend taking a moment to write down what a successful deployment looks like. To start, ask yourself and your team what you want to achieve.
+
+Put another way, what are your desired outcomes?
+
+This takes a small amount of time up front. But, we’ve seen the difference this makes in successful projects. 
+
+Below, we’ve added some key questions to ask yourself, your team, and any other stakeholders.
+
+- Why have we decided to deploy the Mattermost mobile application?
+- Who will, or should, play a key/critical role in this project?
+- Who does this project impact?
+- How much time do we have for this deployment?
+- Should, or could, an initial deployment take a simpler path?
+- Can we stagger our deployment, starting with simple requirements first?
+- Are you going to  run a pilot deployment, if so who with?
+- What  IT or HR policies that need to change?
+
+We recommend setting at least one, achievable and time-based outcome.
+
+At the same time, you should begin thinking about technical and security requirements. A great place to start is with an implementation plan. So, feel free to `use our template available <https://docs.mattermost.com/getting-started/implementation_plan.html>`_ in the Mattermost Documentation. Also, begin by engaging with your technical and security questions like those below.
+
+- Are there any known security or access requirements?
+- Is a VPN connection required?
+- Do we use an Enterprise Mobile Management (EMM) Provider?
+
+Again, this means pausing before you jump into your Mattermost mobile deployment. This takes a little more time, but will make the difference in ensuring your see a return on your investment.
+
+----
+
+###################################
+Mattermost Mobile Deployment Primer
+###################################
+
+A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
+
+We’re here to support you.
+
+This primer serves as an introduction to Mattermost Mobile technologies. It is a compliment to our technical documentation, and focuses on three main areas:
+
+- Technical Requirements
+- Basic Features and Technologies
+- Trouble Spots
+
+Regardless of your expertise, we recommend reading this section before starting your deployment.
```

Comment:
```suggestion
Regardless of your level of expertise, we recommend reading this section before starting your deployment.
```

Reply:
What’s the difference? If there’s a problem with that. I’d strike the introductory clause (from the comma back) and make the sentence, “We recommend...”

Reply:
I agree with removing "Regardless of your expertise". To answer why I included "level": "Level" refers to a variation in expertise which I believe this sentence is trying to convey. "Expertise" on its own refers to a mastery of a field/industry etc. To expand on this: "level of expertise" is a range of experience in this specific subject while "expertise" is mastery of a specific field. 

If we're talking about readers' expertise in this subject I recommend "level". If we're talking about "whatever field within which your experience lies" then we don't need "level".

Reply:
Can you point me to some guidance/rules not this? Both seem ambiguous.

Reply:
My primary reference is Google and the different definitions. "Expertise refers to the in-depth knowledge and skill gained through training, practice and, above all, years of practice in a particular area or domain." vs. "level of expertise" which covers varying levels of said expertise.

My initial interpretation of the sentence is that we're covering role-players who range from beginner to expert so we want to refer to their levels vs. mastery. But I also see the merits of just using "expertise" and am 0/5 on my previous comment. :)

Reply:
I'd keep as is.